src/MSAL.PCL/ClientApplicationBase.cs

```diff
@@ -231,18 +232,25 @@ public IEnumerable<User> Users
             return await handler.RunAsync().ConfigureAwait(false);
         }
 
-        internal virtual AuthenticationRequestParameters CreateRequestParameters(Authenticator authenticator, string[] scope,
+        internal virtual AuthenticationRequestParameters CreateRequestParameters(Authority authority, string[] scope,
             string policy, TokenCache cache)
         {
             return new AuthenticationRequestParameters
             {
-                Authenticator = authenticator,
-                TokenCache =  cache,
+                Authority = authority,
+                TokenCache = cache,
                 Scope = scope.CreateSetFromArray(),
                 Policy = policy,
                 RestrictToSingleUser = this.RestrictToSingleUser,
-                RedirectUri = new Uri(this.RedirectUri)
+                RedirectUri = new Uri(this.RedirectUri),
+                CallState = CreateCallState(this.CorrelationId)
             };
         }
+
+        internal CallState CreateCallState(Guid correlationId)
```

Comment:
Guid is non-nullable.

---
In reply to: [77229841](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/116#discussion_r77229841) [](ancestors = 77229841)